lib/util/equation_processor.dart

```diff
@@ -1,117 +1,93 @@
 import "dart:collection";
 
-import 'calculation_handler.dart';
-import 'dartulator_strings.dart';
+import 'calculation.dart'
+    show Calculation, UnaryCalculation, BinaryCalculation;
+import 'dartulator_strings.dart' show parenthesis_open, parenthesis_close;
 
 class EquationProcessor {
 
   // modified from https://www.tutorialspoint.com/javaexamples/data_intopost.htm
-  static process(List input) {
-    Queue<String> operators = new Queue();
+  static process(List equationList) {
+    Queue calculations = new Queue();
```

Comment:
As a fellow Java dev, I share that need, but as an aspiring Dart/web dev I am opening myself up to the new possibilities. 

Already considered! I am thinking of doing an EquationItem class, Calculation would implement this and the parentheses and numbers would fall under subset class types of this as well. Still trying to figure out a good design for all of this that would keep the code readable while also setting up good guidelines for future use.